src/datasource/dsService.js

```diff
@@ -0,0 +1,12 @@
+import DSBridge from './dsBridge';
+
+export default {
```

Comment:
you're using a named import above, so this should be
`export const DataSourceService = { ... }`